src/Autohide.js

```diff
@@ -94,6 +95,7 @@ define(function (require, exports, module) {
             removeSidebarPlusTransition();
             hidePlaceholder();
             OutlineManager.showOutline();
+            $("#outline-list").scrollTop(scrollBackup);
```

Comment:
Does it make sense to check if `scrollBackup` is set to a value before this call or is jQuery taking care of that?

Reply:
I think it is never read without having been previously saved, due to Autohide's own operation.

But I think checking it does not hurt.